language/Portuguese_Brazil/strings.po

```diff
@@ -1688,19 +1691,19 @@ msgstr "Sequência inválida inserida"
 #, c-format
 msgctxt "input-name"
 msgid "%1$s Analog"
-msgstr ""
+msgstr "Analógico %1$s"
 
 #: src/frontend/mame/ui/inputmap.cpp:525
 #, c-format
 msgctxt "input-name"
 msgid "%1$s Analog Inc"
-msgstr ""
+msgstr "Incrementar Analógico %1$s"
 
 #: src/frontend/mame/ui/inputmap.cpp:526
 #, c-format
 msgctxt "input-name"
 msgid "%1$s Analog Dec"
-msgstr ""
+msgstr "Decrementar Analógico %1$s"
```

Comment:
Have you checked the input menus with the length of these strings?  (Check the “Input (this Machine)” menu with a system with analog axes, e.g. `starwars`.)  It’s probably fine, but it’s worth checking for the increased length.

Reply:
Changed them both.